features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -491,24 +491,37 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
 
         if(isEmpty(window.location.hash)){
 
-            if(urlSearchParams.get(ApplicationManagementConstants.APP_STATE_STRONG_AUTH_PARAM_KEY) !==
+            if(urlSearchParams.get(ApplicationManagementConstants.APP_STATE_STRONG_AUTH_PARAM_KEY) ===
                 ApplicationManagementConstants.APP_STATE_STRONG_AUTH_PARAM_VALUE) {
-                handleDefaultTabIndexChange(defaultTabIndex);
+                // When application selection is done through the strong authentication flow.
+                const signInMethodtabIndex: number = renderedTabPanes?.findIndex(
+                    (element: {"componentId": string}) =>
+                        element.componentId === ApplicationManagementConstants.SIGN_IN_METHOD_TAB_URL_FRAG
+                );
+
+                if (signInMethodtabIndex !== -1) {
+                    handleActiveTabIndexChange(signInMethodtabIndex);
+                }
 
                 return;
-            }
+            } else if (urlSearchParams.get(ApplicationManagementConstants.IS_PROTOCOL) ===
+                ApplicationManagementConstants.APP_STATE_STRONG_AUTH_PARAM_VALUE) {
```

Comment:
Shall we define a separate constant or just use the string "true" here, as the term `APP_STATE_STRONG_AUTH_PARAM_VALUE` doesn't fit here?

Reply:
Replaced with "true" from [f2b0eb3](https://github.com/wso2/identity-apps/pull/6241/commits/f2b0eb3a4cf72c96ec9e0dd92c5a8f4dc2a01361)